source/PlatformTracker.hx

```diff
@@ -0,0 +1,36 @@
+package;
+
+import flixel.FlxG;
+import flixel.FlxObject;
+import flixel.FlxSprite;
+import flixel.system.FlxAssets.FlxGraphicAsset;
+import flixel.util.FlxColor;
+import flixel.input.keyboard.FlxKey;
+
+/**
+ * ...
+ * @author Jess Geiger
+ */
+
+ class PlatformTracker extends Player
```

Comment:
You're right, it was unnecessary. I was copying the class declarations of the other classes and assumed that in Haxe, everything had to inherit from one of the top classes (FlxSprite, FlxState, etc). Rookie mistake, and it has been corrected.